services/worker/src/worker/dtos.py

```diff
@@ -222,6 +222,7 @@ class DatasetHubCacheResponse(TypedDict):
     viewer: bool
     partial: bool
     num_rows: int
+    tags: list[str]
```

Comment:
let's use `Literal["croissant", "webdataset"]` (let's create a dedicate type for that)